cli/cmd/org/rename.go

```diff
@@ -114,6 +95,9 @@ func RenameCmd(cfg *config.Config) *cobra.Command {
 			return nil
 		},
 	}
+	renameCmd.Flags().SortFlags = false
+	renameCmd.Flags().StringVar(&name, "org", cfg.Org, "Current Org Name")
```

Comment:
Setting defaults here is not required since we are always taking input from the user if flag is not set.

Reply:
keeping a default value as it will show up in the prompt if user has not explicitly provided
and user can just enter for default one